lib/Control/Monad.hm

```diff
@@ -13,7 +13,7 @@
 --
 -----------------------------------------------------------------------------
 module Control.Monad
-  ( class Monad, bind, (>>=), return
+  ( class Monad, bind, (>>), (>>=), return
```

Comment:
https://github.com/idris-lang/Idris2/pull/729

I don't think it's a good idea to invoke the name, it would be better to leave it for Control.Arrow